packages/contracts/src/package.json

```diff
@@ -16,8 +16,12 @@
   },
   "homepage": "https://github.com/aragon/osx#readme",
   "dependencies": {
+    "@aragon/osx-commons-contracts": "1.4.0-alpha.1",
     "@ensdomains/ens-contracts": "0.0.11",
-    "@openzeppelin/contracts": "4.9.3",
-    "@openzeppelin/contracts-upgradeable": "4.9.3"
+    "@openzeppelin/contracts": "4.9.5",
+    "@openzeppelin/contracts-upgradeable": "4.9.5"
+  },
+  "devDependencies": {
+    "@aragon/osx-commons-sdk": "0.0.1-alpha.2"
   }
```

Comment:
why does the contracts-only package need the SDK as a dev dependency?

Reply:
To have the helper functions, permission IDs, events, etc. that the osx-commons-sdk offers for testing.

Reply:
then this dev dep should be in `package/contracts/package.json` and not `package/contracts/src/package.json`. The solidity files in `src` don't need the SDK for helper functions

Reply:
Oh, true. It is already present in `package/contracts/src/package.json`. Removed in ee11e74898dedac7e3d0fdc3d54774b4b007cb7f